pkg/controller/job/controller_test.go

```diff
@@ -161,7 +161,7 @@ func TestControllerSyncJob(t *testing.T) {
 		"too few active pods, with controller error": {
 			2, 5,
 			fmt.Errorf("Fake error"), 0, 1, 1, 0,
-			0, 0, 1, 1, 0, false,
+			1, 0, 1, 1, 0, false,
```

Comment:
...but this is a change in the number of expected creations? Why would that change?